target/linux/ramips/mt7621/base-files/etc/board.d/02_network

```diff
@@ -42,6 +42,10 @@ ramips_setup_interfaces()
 	mikrotik,rb750gr3)
 		ucidef_set_interfaces_lan_wan "lan2 lan3 lan4 lan5" "wan"
 		;;
+	zyxel,wap6805)
+		ucidef_set_interface_lan "lan0 lan1 lan2 lan3"
```

Comment:
Are they labelled as lan0..3 instead of lan1..4 ?

Reply:
No, they're not.  This I did intend to look up before submitting, but forgot..The ports are labelled LAN1 to LAN4, where port on the switch 0 is LAN4.  Fixed now.

Thanks a lot for the review!